src/backend/distributed/commands/utility_hook.c

```diff
@@ -535,6 +535,7 @@ static void
 ExecuteDistributedDDLJob(DDLJob *ddlJob)
 {
 	bool shouldSyncMetadata = ShouldSyncTableMetadata(ddlJob->targetRelationId);
+	int targetPoolSize = ddlJob->executeSequentially ? 1 : 4;
```

Comment:
Maybe this PR is a good one to make 4 a constant? Then in future if we want to make this a GUC we will have an easier time finding all occurrences of it.

Reply:
I don't want to it just right now because we might pick a different API such that instead of exposing `targetPoolSize ` we might consider something else. See some [ideas](https://docs.google.com/document/d/1MuJ6V6E7X_25J4kFN0uXFiFwLMcq8h4I0PGJd-XqqKA/edit?usp=drive_web&ouid=103285309091993434178), which requires some more discussion.

Does that sound OK to you?